Sources/examples-cli/Commands/AlertCommand.swift

```diff
@@ -0,0 +1,26 @@
+import ArgumentParser
+import Foundation
+import Noora
+
+struct AlertCommand: AsyncParsableCommand {
+    static let configuration = CommandConfiguration(
+        commandName: "alert",
+        abstract: "A command that showcases the alert components."
+    )
+
+    func run() async throws {
+        Noora().warning([
+            ("Your token is about to expire", next: "Generate a new token with \(.command("tuist project tokens create"))"),
+        ])
```

Comment:
As mentioned above, I'd aim to simplify this to `Noora().warning("Your token is about to expire", nextSteps: "...")`